File: translation_output/wizardcoder/codenet/python/java-sanitized/p03371.java
import java.util.*;

public class p03371 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int A, B, C, X, Y;
        A = sc.nextInt();
        B = sc.nextInt();
        C = sc.nextInt();
        X = sc.nextInt();
        Y = sc.nextInt();

        int mi = Math.min(X, Y);
        int a = A * X + B * Y;
        int b = mi * C * 2 + A * (X - mi) + B * (Y - mi);
        int c = Math.max(X, Y) * C * 2;

        System.out.println(Math.min(a, Math.min(b, c)));
    }
}

File: Results/DER/Translation/translation_output/misleading_test/gpt-3.5/codenet/java/python/p03371.py
```python
import sys

class Scanner:
    def __init__(self, input_source=sys.stdin):
        self.input_source = input_source
        self.tokens = iter(input_source.read().split())

    def next(self):
        return next(self.tokens)

    def nextInt(self):
        return int(self.next())

def main():
    in_stream = Scanner()
    a = in_stream.nextInt()
    b = in_stream.nextInt()
    c = in_stream.nextInt()
    x = in_stream.nextInt()
    y = in_stream.nextInt()

    if a + b < 2 * c:
        print(x * a + y * b)
    else:
        mini = min(x, y)
        maxi = max(x, y)
        res = 0
        res += 2 * c * mini
        if maxi == x:
            res += min(a, 2 * c) * (maxi - mini)
        else:
            res += min(b, 2 * c) * (maxi - mini)
        print(res)

if __name__ == "__main__":
    main()
```

